org.kde.kst.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>org.kde.kst.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL</project_license>
+  <name>kst</name>
+  <summary>Interactive Data Visualization and Analysis</summary>
+  <description>
+    <p>
+      Kst is the fastest real-time large-dataset viewing and plotting tool available and has built-in data analysis functionality.
+    </p>
+  </description>
+  <url type="homepage">https://https://kst-plot.kde.org/</url>
+  <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?product=kst&amp;format=guided</url>
+  <project_group>KDE</project_group>
+  <provides>
+    <binary>kst2</binary>
+  </provides>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="2.0.81" date="2022-05-02"/>
+    <release version="2.0.8" date="2014-07-20"/>
+  </releases>
```

Comment:
there is no screenshot (mandatory)

Reply:
I am about to write a PR for gettings this merged in the upstream repo, my plan was to make a commit that removes the AppData file from here when it has been merged. Is this a blocker? I will fix the screenshots